explainer.md

```diff
@@ -322,6 +321,26 @@ cancelable.
     that continues allowing a permission on a
     [previously granted](#ui-when-the-permission-is-already-granted) type of
     UI).
+-   `onvalidationstatuschange` - raised when the PEPC switches from being
+    "valid" to "invalid". The PEPC is considered "valid" when the user agent
+    trusts the integrity of the signal if the user were to click to click on it,
+    and "invalid" otherwise. There are many reasons for which an element can
+    become "invalid" as detailed in the [Security](#security) section, but to
+    enumerate a few: element style is invalid, element is covered, element has
+    recently moved, element has changed size, element is not fully visible in
+    the viewport, etc. A new `Event` subclass will be added,
+    `ValidationStatusChangeEvent`, which contains two extra fields:
+    - `boolean isValid` - indicates whether the status has transitioned to
```

Comment:
I think `isValid` should be on the `<permission>` element, then it's just a regular event.

Reply:
I've moved both of them to be attributes, which makes more sense indeed. Thank you.